clusterloader2/testing/list/config.yaml

```diff
@@ -0,0 +1,189 @@
+# List benchmark
+{{$namespaceNumber := DefaultParam .CL2_LIST_NAMESPACE_NUMBER 4}}
+{{$configMapBytes := DefaultParam .CL2_LIST_CONFIG_MAP_BYTES 10000}}
+{{$configMapGroup := DefaultParam .CL2_LIST_CONFIG_MAP_GROUP "list-configmap"}}
+{{$configMapNumber := DefaultParam .CL2_LIST_CONFIG_MAP_NUMBER 3000}}
+{{$secretBytes := DefaultParam .CL2_LIST_SECRET_BYTES 10000}}
+{{$secretGroup := DefaultParam .CL2_LIST_SECRET_GROUP "list-secret"}}
+{{$secretNumber := DefaultParam .CL2_LIST_SECRET_NUMBER 3000}}
+
+{{$listReplicas := DefaultParam .CL2_LIST_BENCHMARK_PODS 1}}
+{{$inflight := DefaultParam .CL2_LIST_CONCURRENCY 2}}
+{{$qps := DefaultParam .CL2_LIST_QPS -1}}
+
+{{$namespacedPodCpu := DefaultParam .CL2_LIST_BENCHMARK_POD_CPU "400m"}}
+{{$namespacedPodMemory := DefaultParam .CL2_LIST_BENCHMARK_POD_MEMORY "800Mi"}}
+{{$clusterScopedPodCpu := DefaultParam .CL2_LIST_BENCHMARK_POD_CPU "2000m"}}
+{{$clusterScopedPodMemory := DefaultParam .CL2_LIST_BENCHMARK_POD_MEMORY "4000Mi"}}
+
+name: List benchmark
+namespace:
+  number: {{$namespaceNumber}}
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 10
+steps:
+- name: Setup namespace for list benchmark pods
+  phases:
+  - replicasPerNamespace: 1
+    tuningSet: Sequence
+    objectBundle:
+    - basename: list-benchmark
+      objectTemplatePath: namespace.yaml
+- name: Setup permissions
+  phases:
+  - replicasPerNamespace: 1
+    tuningSet: Sequence
+    objectBundle:
+    - basename: list-clusterrole
+      objectTemplatePath: clusterrole.yaml
+  - replicasPerNamespace: 1
+    tuningSet: Sequence
+    objectBundle:
+    - basename: list-clusterrolebinding
+      objectTemplatePath: clusterrolebinding.yaml
+  - namespaceRange:
+      min: 1
+      max: {{$namespaceNumber}}
+    replicasPerNamespace: 1
+    tuningSet: Sequence
+    objectBundle:
+    - basename: list-rolebinding
+      objectTemplatePath: rolebinding.yaml
+
+- name: Create configmaps
+  phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaceNumber}}
+    tuningSet: Sequence
+    replicasPerNamespace: {{$configMapNumber}}
+    objectBundle:
+    - basename: {{$configMapGroup}}
+      objectTemplatePath: configmap.yaml
+      templateFillMap:
+        bytes: {{$configMapBytes}}
+        group: {{$configMapGroup}}
+- name: Create secrets
+  phases:
+  - namespaceRange:
+      min: 1
+      max: {{$namespaceNumber}}
+    tuningSet: Sequence
+    replicasPerNamespace: {{$secretNumber}}
+    objectBundle:
+    - basename: {{$secretGroup}}
+      objectTemplatePath: secret.yaml
+      templateFillMap:
+        bytes: {{$secretBytes}}
+        group: {{$secretGroup}}
```

Comment:
might wanna combine these two phases into a single step - they can happen in parallel

Reply:
Done